packages/lit-analyzer/package.json

```
@@ -25,15 +25,55 @@
 	},
 	"scripts": {
 		"clean": "rimraf lib",
-		"prepare": "npm run build",
-		"build": "npm run clean && npm run build:ts && npm run check-version",
-		"build:ts": "tsc",
-		"watch": "tsc --watch",
+		"build": "wireit",
 		"readme": "readme generate -i readme.blueprint.md -c readme.config.json",
-		"test": "ava --color",
-		"test:all": "ava --color --fail-fast",
+		"test": "wireit",
+		"test:ava": "wireit",
+		"test:all": "npm test",
 		"test:watch": "ava --watch --color",
-		"check-version": "node scripts/check-version.js"
+		"check-version": "wireit"
+	},
+	"wireit": {
+		"build": {
+			"command": "tsc --build --pretty",
+			"files": [
+				"src/**/*",
+				"tsconfig.json"
+			],
+			"output": [
+				"lib",
+				"test",
+				"scripts",
+				"index.js",
+				"index.d.ts",
+				"index.d.ts.map",
+				"./.tsbuildinfo"
+			],
+			"clean": "if-file-deleted"
+		},
+		"test": {
+			"dependencies": [
+				"test:ava",
+				"check-version"
+			]
+		},
+		"test:ava": {
+			"dependencies": [
+				"build"
+			],
+			"command": "ava --color",
+			"output": [],
+			"files": []
+		},
+		"check-version": {
+			"dependencies": [
+				"build"
+			],
+			"files": [
+				"package.json"
```

Assuming this is not a build artifact:

```suggestion
				"package.json",
				"scripts/check-version.js"
```